.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  pull_request:
+
+env:
+  FASTLANE_SKIP_UPDATE_CHECK: true
+  FASTLANE_XCODE_LIST_TIMEOUT: 30
+  MATCH_KEYCHAIN_NAME: "gh_actions"
+
+jobs:
```

Comment:
is there any way to cache the bundler and cocoapods dependencies?

Reply:
Already done, just wasn't pushed 🙂